### DISCUSSION TITLE
Patterns -  tips and tricks for developers

### DISCUSSION THREAD

**Discussion by bph** [WordPress/developer-blog-content#226] [Topic Ideas]
Now that patterns are a massive component of block theme development, and theme developers might want to reuse certain patterns over multiple sites/theme, it would be good to know what tips designers can give who have published a series of block themes. What to think about when you want to create site-agnostic patterns? 

What makes patterns most manageable ( wrap it in  a top-level group block )
How best to use styles? What about colors and typography? 

How to handle content and media assets?

**Comment by justintadlock** [comment]
Note that there's a lot of cross-over with this topic and one that is already approved: https://github.com/WordPress/developer-blog-content/issues/214

I've written a good chunk of it, but got a little stuck and haven't revisited it yet: https://docs.google.com/document/d/1TVSeEklV2StuUmwwIUu_1_SscQtc1XZ8jVMbltC05fM/edit?usp=sharing

Was there already a specific writer for this proposal? I thought we had someone. If so and we want to combine these two topics, I'm happy to let them take over the work that I've already started (I always have plenty of other topics to tackle).

**Reply by justintadlock** [reply]
@beafialho tagging you here. @bph was kind enough to remind me you were going to do this.

I'm happy to keep the topics separate or combine them, depending on what you have in mind.

**Comment by justintadlock** [comment]
This is now approved and a ticket has been created here: https://github.com/WordPress/developer-blog-content/issues/235

**Comment by bph** [comment]
Published [Pattern design tips and tricks for developers](https://developer.wordpress.org/news/2024/03/19/pattern-design-tips-and-tricks-for-developers/)
